docs/conf.py

```diff
@@ -34,6 +43,11 @@
 # The name of the Pygments (syntax highlighting) style to use.
 pygments_style = 'sphinx'
 
+# -- Options for extlinks extension ---------------------------------------
+extlinks = {
+    'user': (f'{github_sponsors_url}/%s', '@'),  # noqa: WPS323
```

Comment:
Is sponsors the right URL for a user? Why not `/user/%s`?

Reply:
If sponsors is activated for the given user, it'll stop there, if not it'll be redirected to the normal user page.

Reply:
SGTM